src/plugins/intel_gpu/include/intel_gpu/plugin/common_utils.hpp

```diff
@@ -111,6 +112,21 @@ inline ov::Shape predict_shape(const std::string& name, const cldnn::layout layo
     return layout.get_shape();
 }
 
+inline cldnn::memory::ptr allocate_memory_evenif_zero_bytes(cldnn::engine& _engine, const cldnn::layout& layout, cldnn::allocation_type type, bool reset = true) {
+    if (layout.bytes_count() == 0) {
+        auto non_zero_layout = cldnn::layout({1}, layout.data_type, layout.format);
+        auto res = _engine.allocate_memory(non_zero_layout, type, reset);
```

Comment:
I think we can ignore reset value here in case of "empty buffer"

Reply:
Updated.